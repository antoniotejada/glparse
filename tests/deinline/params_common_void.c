//
// Verify that functions that are left with no parameters work and are left with
// a void parameter
//

void f1(int param_int_0)
{
    a(1);
    b(1);
    b(1);
    b(1);
    b(1);
    b(1);
    b(1);
    c(1);
    d(1);
}

void f2(int param_int_0)
{
    a(1);
    b(1);
    c(1);
    c(1);
    c(1);
    c(1);
    c(1);
    c(1);
    d(1);
}