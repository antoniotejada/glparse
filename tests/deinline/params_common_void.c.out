// Src lines 18
// Dest lines 16
//
// Verify that functions that are left with no parameters work and are left with
// a void parameter
//

void f1(int param_int_0);
void f2(int param_int_0);
void subframe2();
void subframe3();

void f1(int param_int_0)
{
    a(1);
    subframe2();
    subframe2();
    c(1);
    d(1);
}

void f2(int param_int_0)
{
    a(1);
    b(1);
    subframe3();
    subframe3();
    d(1);
}

void subframe2()
{
    b(1);
    b(1);
    b(1);
}

void subframe3()
{
    c(1);
    c(1);
    c(1);
}
